.github/workflows/create_credentials_requirements_pr.yml

```diff
@@ -58,7 +58,7 @@ jobs:
               base: 'master',
             })
             // Get the PR that relates to the tagged commit
-            const getRelatedPrResponse = await github.rest.repos.listPullRequestsAssociatedWithCommit({
+            const getRelatedPrResponse = await github.rest.repos.listPullRequestsAssociatedWithCommit.endpoint.merge({
```

Comment:
I'm not super familiar with the GitHub API and how this is used, but I'm okay with giving this a try since the workflow is already broken. We can always revert if it doesn't work.